tools/docs/manager.ts

```diff
@@ -152,7 +152,7 @@ sidebar_label: ${displayName}
           md += '\n```\n\n';
         }
       }
-      md += `For details on how to extend a manager's \`fileMatch\` value, please follow [this link](/modules/manager/#file-matching).\n\n`;
```

Comment:
This might not be necessary given the other changes.

Reply:
I think let's revert this change